Model/Config/Source/ThreeDSFlow.php

```diff
@@ -15,8 +15,8 @@
 
 class ThreeDSFlow implements OptionSourceInterface
 {
-    const THREEDS_NATIVE = 'native';
-    const THREEDS_REDIRECT = 'redirect';
+    const THREEDS_NATIVE = 'preferred';
+    const THREEDS_REDIRECT = 'disabled';
```

Comment:
Would it be a good idea to also change the const names to reflect the value better? Just to minimize confusion

Reply:
I did gave it a thought, but the Possible values:
preferred: Use native 3D Secure authentication when available.
disabled: Only use the redirect 3D Secure authentication flow.

Reply:
Hey @khushboo-singhvi, these are the constants of a public class. Modifying the values might introduce a breaking change. Besides that, the values are used in the configuration table. The changes in the values require data patch for `core_config_data`.